### PR TITLE
Feature/94 azure pipelines cd

### PR DESCRIPTION
**Pull request by alex-flores-jalafund**
Feature/94 azure pipelines cd

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -11,34 +11,82 @@ pr:
...
-    ArtifactName: 'Drop'
-    publishLocation: 'Container'
-  
\ No newline at end of file
+variables:
+  projects: '**/*.sln'
+  azureSubscription: 'devlevel3-dev (a797242d-ce60-41aa-98ca-2e22d5221cfe)'
+  functionAppName: 'fa-tuapi-resumes-dev-bra'
+  vmImageName: 'ubuntu-latest'
+
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml:19]: we don't want variables for this pipeline, please remove all and keep the values, we'll add them through Azure DevOps UI

**Review by paulo-teran-jalafund — DISMISSED** [review]
LGTM

File: azure-pipelines.yml [hunk 2]
```diff
@@ -11,34 +11,82 @@ pr:
...
+stages:
+- stage: Build
+  displayName: Build & Test
+
+  jobs:
+  - job: Build
+    displayName: BuildJob
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml]: remove displayName here

File: azure-pipelines.yml [hunk 3]
```diff
@@ -11,34 +11,82 @@ pr:
...
+        command: 'build'
+        projects: $(projects)
+        arguments: '--output $(Build.ArtifactStagingDirectory) $(styleCopEnabled)'
+
+    - task: DotNetCoreCLI@2
+      displayName: 'DotNet Tests'
+      inputs:
+        command: test
+        projects:  $(testsProjects)
+    - task: ArchiveFiles@2
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml:47]: add a space between tasks for me please

Comment (martin-callejas-jalafund) [reply azure-pipelines.yml:47]: add displayName: 'Archive files'

File: azure-pipelines.yml [hunk 4]
```diff
@@ -11,34 +11,82 @@ pr:
...
+      # condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+      inputs:
+        PathtoPublish: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
+        ArtifactName: 'drop'
+        publishLocation: 'Container'
+
+- stage: Deploy
+  displayName: Deploy stage
+  dependsOn: Build
+  condition: succeeded()
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml]: make it work **only for** develop and master merges, we don't want deploy everytime

**Review by martin-callejas-jalafund — CHANGES_REQUESTED** [review]
Good job! Soon we will centralize the backend pipelines to avoid duplicated code.

File: azure-pipelines.yml (hunk 1 above)
Comment (alex-flores-jalafund) [reply azure-pipelines.yml:19]: It's done

File: azure-pipelines.yml (hunk 2 above)
Comment (alex-flores-jalafund) [reply azure-pipelines.yml]: It's done

File: azure-pipelines.yml (hunk 3 above)
Comment (alex-flores-jalafund) [reply azure-pipelines.yml:47]: It's done

File: azure-pipelines.yml (hunk 4 above)
Comment (alex-flores-jalafund) [reply azure-pipelines.yml]: It's done

**Review by fernando-maceda-jalafund — DISMISSED** [review]
LGTM

**Review by rodrigo-baldiviezo-jalafund — DISMISSED** [review]

**Review by fernando-maceda-jalafund — DISMISSED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (martin-callejas-jalafund) [reply azure-pipelines.yml:19]: no, it's not done, I wanted them to be added trough Az UI, but now you hardcoded them/
it's fine, let's iterate to enhance

**Review by martin-callejas-jalafund — APPROVED** [review]

**Review by fernando-maceda-jalafund — APPROVED** [review]
LGTM

**Review by paulo-teran-jalafund — APPROVED** [review]
LGTM

**Review by rodrigo-baldiviezo-jalafund — APPROVED** [review]

**Pull request merged**